Add a Sponsor Expo that combines Job Fair & Product demos (#1928)
writethedocs/www: docs/conf/portland/2023/job-fair.rst

:template: {{year}}/generic.html

Sponsor Expo
============

This year we're inviting our top sponsors to engage with our attendees at our Sponsor Expo.

> This is a combination of two primary events:

Reviewer: ```suggestion
This is a combination of two things:
```

primary? probably just s/primary// ?

> * Our job fair, where companies can talk with attendees who are interested in work
> * An area for product demos, for companies building products that our audience might use

Reviewer: ```suggestion
* A job fair, where companies can talk with attendees who are interested in work
* Product demos, for companies building products that our audience might use
```

> We're excited to invite you to mingle with our top sponsors in this area,
> and learn more about what opportunities they have.

Reviewer: We use a lootta excited

```suggestion
We're looking forward to facilitating constructive networking, connecting people with jobs and employers, and hearing more from some of our sponsors. 
```
